org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/SonarLintCorePlugin.java

```diff
@@ -114,6 +140,21 @@ public void start(BundleContext context) {
     super.start(context);
     sonarLintChangeListener = new SonarLintChangeListener();
     ResourcesPlugin.getWorkspace().addResourceChangeListener(sonarLintChangeListener, IResourceChangeEvent.POST_CHANGE);
+
+    modulePathManager = new ModulePathManager();
+    trackingChangeQueueManager = new TrackingChangeQueueManagerImpl();
+    trackingChangeQueueManager.subscribe(new MarkerUpdater(modulePathManager));
+
+    IssueTrackerCacheFactory factory = localModuleKey -> {
+      // TODO find a better way to get to the .settings dir of an Eclipse project
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1135)